### PR TITLE
Setup enterprise docs

### PR DESCRIPTION
**Pull request by ned1313**
This update creates an expanded setup section with separate docs for Kind, Installation, and Cluster Registration. It also includes a reference doc for the RBAC policy applied by Gloo Mesh in the cluster role for registered clusters.

Addresses issue #1023

### CODE REVIEW

File: docs/content/reference/cluster_role.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: "Registered ClusterRole Definition"
+menuTitle: Registered ClusterRole
+description: ClusterRole created when registering a cluster with Gloo Mesh.
+weight: 10
+---
+
+The following YAML shows the ClusterRole created on a target cluster when it is registered with Gloo Mesh.
```
[the remark this answers is not in the document]
Comment (ned1313) [reply docs/content/reference/cluster_role.md:8]: Updated

**Review by joekelley — APPROVED** [review]

**Pull request merged**